internal/aez/ccd_power_regulators.go

```diff
@@ -54,6 +57,7 @@ type CPRU struct {
 
 //CPRUReport structure
 type CPRUReport struct {
+	STAT
```

Comment:
Do we really want the STAT as it is in the Report? It seems we want a STATReport here with processed readable data based on the stats info.